protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -11,7 +11,7 @@ When trading Spot products, parties can only use assets they own - there is no l
 
 ## 2. Network Parameter
 
-1. `spot_trading_enabled`: parameter defines whether markets using Spot products are enabled on the network.
+1. `SpotMarketTradingEnabled`: parameter defines whether markets using Spot products are enabled on the network.
```

Comment:
```suggestion
1. `limits.markets.proposeSpotEnabledFrom`: parameter defines whether markets using Spot products are enabled on the network.
```

@ze97286 - we have reviewed this in the protocol design meeting and suggest the net param should be named as suggested here

Reply:
proposeSpotEnabledFrom, is this a date?

Reply:
Good point! I think it should be `limits.markets.proposeSpotEnabled` and it should just be one boolean. 

I don't think we want to bother with the "enabled from" date.

Reply:
suggestion updated - can be reviewed and committed in todays meeting